showcase/showcase-components/plots-showcase.js

```diff
@@ -0,0 +1,4 @@
+import LineSeriesCanvas from './plot/line-series-canvas';
```

Comment:
Would you mind putting your component in the main plot showcase instead? see https://github.com/uber/react-vis/blob/master/showcase/showcase-sections/plots-showcase.js